.github/workflows/linting.yml

```
@@ -37,3 +36,17 @@ jobs:
         uses: jbergstroem/hadolint-gh-action@v1
         with:
           dockerfile: "./docker/Dockerfile*"
+
+  run-python-lint:
+    name: python-lint
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)